src/scripts/modules/media/body/content-baked.less

```diff
@@ -278,10 +278,20 @@ h1.example-title .text {
 }
 .os-eoc [data-type="exercise"],
 .os-eoc .exercise {
-  p { display: inline; }
+  p {display: inline; }
   [data-type="problem"] > .number {
     .exercise-number();
   }
+
+  img {
+    float: left;
+    margin-right: 10px;
+  }
+
+  ol {
+    overflow: auto;
```

Comment:
Overflow `auto` adds scrollbars when content is overflowing. I think that setting `hidden` there might be better, because we don't want to scroll through elements if this situation arises.

Reply:
@Adrianltd good catch! Can you change your review to change requested instead of approved?

Reply:
@helenemccarron will do